seatunnel-config/src/main/java/com/typesafe/config/impl/SeaTunnelParseableFile.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.typesafe.config.impl;
+
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigOrigin;
+import com.typesafe.config.ConfigParseOptions;
+import com.typesafe.config.ConfigParseable;
+import com.typesafe.config.ConfigSyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
+import java.util.Iterator;
+
+public class SeaTunnelParseableFile extends Parseable {
+    private final File input;
+
+    public SeaTunnelParseableFile(File input, ConfigParseOptions options) {
+        this.input = input;
+        postConstruct(options);
+    }
+
+    @Override
+    protected Reader reader() throws IOException {
+        if (ConfigImpl.traceLoadsEnabled()) {
+            trace("Loading config from a file: " + input);
+        }
+        InputStream stream = new FileInputStream(input);
+        return readerFromStream(stream, "UTF-8");
```

Comment:
"UTF-8" can  be replaced with StandardCharsets.UTF_8

Reply:
Make sense.